<launch>
  <arg name="use_cuda" default="true" />
  <arg name="threshold" default="0.5" />
  <arg name="template_dir" default="$(find online_data_generator)/templates" />
  <arg name="input_image" default="/remote/openni_camera/rgb/image_rect_color" />

  <node name="image_publisher"
        pkg="jsk_perception" type="image_publisher.py"
        clear_params="true">
    <rosparam subst_value="true">
      file_name: $(find online_data_generator)/data/sample1.jpg
      rate: 30
    </rosparam>
  </node>

  <node name="deep_conv_feature_template_matching"
        pkg="online_data_generator" type="deep_conv_feature_template_matching.py"
        output="screen">
    <rosparam subst_value="true">
      use_cuda: $(arg use_cuda)
      threshold: $(arg threshold)
      template_dir: $(arg template_dir)
    </rosparam>
    <!-- <remap from="~input" to="image_publisher/output" /> -->
    <remap from="~input" to="$(arg input_image)" />
  </node>

</launch>
